### PR TITLE
Update Redis example to use Tailwind.

### PR DESCRIPTION
**Pull request by leerob**
And an improved design 😄 

https://roadmap-redis.vercel.app/

### CODE REVIEW

File: examples/with-redis/package.json [hunk 1]
```diff
@@ -1,19 +1,26 @@
 {
   "name": "with-redis",
-  "version": "0.1.1",
+  "version": "0.1.0",
+  "private": true,
```
Comment (lfades) [inline examples/with-redis/package.json:4]:
```suggestion
  "license": "MIT",
```

File: examples/with-redis/public/upstash.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...0.045-9.945 10.045-26.07 0-36.017l-4.546 4.503a18.968 18.968 0 010 27.012c-7.534 7.46-19.749 7.46-27.282 0L.098 53.118z"/><path fill="#00E9A3" d="M9.192 44.114c5.022 4.973 13.165 4.973 18.188 0a12.645 12.645 0 000-18.008l-4.547 4.502a6.323 6.323 0 010 9.004c-2.512 2.487-6.583 2.487-9.094 0l-4.547 4.502zM45.567 8.097c-10.045-9.946-26.33-9.946-36.376 0-10.045 9.946-10.045 26.071 0 36.017l4.547-4.502a18.968 18.968 0 010-27.013c7.534-7.46 19.748-7.46 27.282 0l4.547-4.502z"/><path fill="#00E9A3" d="M36.473 17.101c-5.022-4.972-13.165-4.972-18.188 0a12.645 12.645 0 000 18.009l4.547-4.502a6.323 6.323 0 010-9.005c2.512-2.486 6.583-2.486 9.094 0l4.547-4.502z"/><path fill="#fff" fill-opacity=".8" d="M45.567 8.097c-10.045-9.946-26.33-9.946-36.376 0-10.045 9.946-10.045 26.071 0 36.017l4.547-4.502a18.968 18.968 0 010-27.013c7.534-7.46 19.748-7.46 27.282 0l4.547-4.502z"/><path fill="#fff" fill-opacity=".8" d="M36.473 17.101c-5.022-4.972-13.165-4.972-18.188 0a12.645 12.645 0 000 18.009l4.547-4.502a6.323 6.323 0 010-9.005c2.512-2.486 6.583-2.486 9.094 0l4.547-4.502z"/></g><defs><clipPath id="clip0"><path fill="#fff" d="M0 0h45.574v60.104H0z" transform="translate(.043 .53)"/></clipPath></defs></svg>
```
Comment (lfades) [inline examples/with-redis/public/upstash.svg:1]: Should this file be removed instead?

**Review by lfades — CHANGES_REQUESTED** [review]
@leerob We can't have live demos that allow for bad language to be introduced. Let's disable the input in the live demo.

**Comment by leerob** [comment]
@lfades I'll add a `bad-words` check for the demo, similar to the MySQL example 👍

**Review by lfades — APPROVED** [review]

**Pull request merged**